internal/pkg/ecs/ecs.go

```diff
@@ -449,10 +450,23 @@ func (c Client) serviceARN(app, env, svc string) (*ecs.ServiceArn, error) {
 	if len(services) == 0 {
 		return nil, fmt.Errorf("no ECS service found with tags %s", tags.String())
 	}
-	if len(services) > 1 {
+	arns := make([]string, len(services))
+	for i := range services {
+		arns[i] = services[i].ARN
+	}
+	serviceARN := ecs.ServiceArn(arns[0])
```

Comment:
is this making the assumption that all of tagged the services are in the same cluster? isn't it possible that's not true?

Reply:
They are in the same cluster i think for this use case. We can make another function or modify this one if we need a different behavior in the other place

Reply:
I think having to specify an app+env provides enough security, right? I don't think Copilot will let you make two envs of the same name in the same app. Though what happens if someone creates an app in tools account A, then an env in account B, then an app in tools account C, and an env in account B? Maybe this is out of scope 😂

Reply:
Actually i'll change the logic